### PR TITLE
Fix/assign people to project redesign

### PR DESCRIPTION
**Pull request by Gaston-Gonzalez**

### CODE REVIEW

File: src/components/AsignacionDialog/index.js [hunk 1]
```diff
@@ -154,7 +154,10 @@ function AsignacionDialog({
                 value={startDate}
                 minDate={moment(startDate)}
                 maxDate={moment(endDate)}
-                onChange={(e) => onInputChange(e, "start_date")}
+                onChange={(e) => {
```
Comment (piarovira1) [inline src/components/AsignacionDialog/index.js:157]: e name is confusing. Let's use something as newDate or date

File: src/components/AsignacionDialog/index.js [hunk 2]
```diff
@@ -181,7 +184,10 @@ function AsignacionDialog({
                 value={endDate}
                 minDate={moment(startDate)}
                 maxDate={moment(endDate)}
-                onChange={(e) => onInputChange(e, "end_date")}
+                onChange={(e) => {
```
Comment (piarovira1) [inline src/components/AsignacionDialog/index.js:187]: Same

File: src/containers/AsignarPersonaAProyecto/index.js [hunk 3]
```diff
@@ -114,9 +114,8 @@ export default function AgregarPersona({
                 projectData.people = [...projectData.people, personaNueva];
                 editRow(projectData);
               }
-
               enqueueSnackbar(
-                `Se asigno el rol ${role} a: ${person.name} en ${projectData.name} con éxito.`,
+                `Se asigno el rol ${role} a ${person.name} en ${projectData.name} con éxito.`,
```
Comment (piarovira1) [inline src/containers/AsignarPersonaAProyecto/index.js:118]: Typo: asignó

File: src/components/CardSelector/index.js [hunk 4]
```diff
@@ -19,41 +22,73 @@ CardSelector.propTypes = {
...
+                      id={value[0]}
+                      checked={value[1]}
+                      tabIndex={-1}
+                      disableRipple
+                      icon={<RadioButtonUncheckedIcon />}
+                      checkedIcon={<CheckCircleIcon />}
+                    />
+                  </ListItemIcon>
+                  <ListItemText primary={value[0]} />
+                </ListItem>
```
Comment (jpmazza) [inline src/components/CardSelector/index.js:87]: This is kind of duplicated? I think it's a good idea creating a sub-component with this. Right?

**Review by jpmazza — APPROVED** [review]

**Pull request merged**